src/files/schemas.ts

```diff
@@ -34,6 +34,7 @@ export const AssetJSONSchema = {
     collectionId: { type: 'string', format: 'uuid' },
     name: { type: 'string' },
     description: { type: 'string' },
+    urn: { type: 'string' },
```

Comment:
wouldn't this type make it so `urn` is required? I mean, that you **need** to supply an urn, at least a `''`

Reply:
As of today, the URNs can be a string, properly formatted or a nulled value (that is taken into consideration to define a DCL collection). I left it as non-required for that specific case (that is, at the end, in the required property, `urn` is not in the list).

Reply:
👍